Rocket.Chat/Helpers/UploadHelper.swift

```diff
@@ -45,6 +46,17 @@ struct UploadHelper {
         )
     }
 
+    /**
+        This method returns the size of some Data object.
+
+        - parameters:
+          - data: The data object to be inspected.
+        - returns: The size of the data
+     */
+    static func sizeFor(_ data: Data) -> Int {
+        return (data as NSData).length
```

Comment:
Seems to be working again - still on FileSystem. 🤔
Also tested directly on my phone: MongoDB size matches the file size. But as I updated the server to 0.60 on 28/12 that may have fixed the problem... So maybe it was just the server messing up.

Reply:
@soundstorm The app is not using WebSockets to send the file when server is 0.60, that's why it works just fine everything, hehe.